src/stars.css

```
@@ -0,0 +1,51 @@
+.stars {
+  padding-bottom: 24px;
+
+  display: flex;
+  flex-direction: row;
+
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--product-name-color);
+}
+
+.stars__elem {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars__elem--1,
+.stars__elem--2,
+.stars__elem--3,
+.stars__elem--4,
+.stars__elem--5 {
+  display: flex;
+  flex-direction: row;
+}
```

no needs to count all cases here
```suggestion
.stars {
  display: flex;
}
```

---

done